packages/office-ui-fabric-react/src/components/Nav/__snapshots__/Nav.test.tsx.snap

```diff
@@ -184,28 +184,6 @@ exports[`Nav renders Nav correctly 1`] = `
                         justify-content: flex-start;
                       }
                 >
-                  <i
```

Comment:
I think that it's good that the tests show that the icon's won't show up if the appropriate information isn't added, but we should probably have one test that ensures the icon is present so we can validate the icons going forward as well.